README.md

```diff
@@ -45,13 +45,16 @@ doing at your convenience.
 
 If you are using CMake and want to use its configuration abilities to save
 you the trouble, you can easily integrate Backward:
+
 ```
-add_subdirectory(/path/to/backward-cpp)
+find_package(Backward)
```

Comment:
does it still allow importing backward from a sub-directory, like `add_subdirectory(/path/to/backward-cpp)` before?